ExodusMovement/exodus-oss
# test: verify feature factory returns a valid shape

I attempted to use the latest keychain in headless, but the feature factory was broken. As I was about to PR my fix, I noticed this is already fixed in #64. Let's keep at least the tests

## Testplan
1. run `yarn add @exodus/argo`
2. run `lerna run todo:reenable:test:integration --scope @exodus/keychain` and verify the feature test passes
